src/components/Notifications/Form/IntegrationRecipientTypeahead.tsx

```diff
@@ -81,11 +82,22 @@ export const IntegrationRecipientTypeahead: React.FunctionComponent<IntegrationR
 
     const onSelect = React.useCallback((_event, value: string | SelectOptionObject) => {
         const integrationSelected = props.onSelected;
+        let validatedState = 'success';
+        const validated = validatedState;
         if (value instanceof RecipientOption) {
             integrationSelected(value);
             setOpen(false);
         }
-    }, [ props.onSelected ]);
+
+        if (selection === undefined) {
+            validatedState = 'error';
+        } else {
+            validatedState = 'success';
+        }
+
+        return validated;
+
+    }, [ props.onSelected, selection ]);
```

Comment:
I don't think this is relevant. IIUC the validation is already in place, we only  need to show it.